### PR TITLE
Feature/apps 2064

### PR DESCRIPTION
**Pull request by MohinishSah**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ on:
       - master
       - 'SP/**'
       - 'HF/**'
+      - 'feature/**'
```
Comment (kavitshah-gl) [inline .github/workflows/build.yml:9]: Do we require this one?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -98,7 +99,7 @@ jobs:
           create-github-annotations: "false"
 
   Release:
-    if: ${{ ( startsWith(github.ref_name, 'HF/')  || startsWith(github.ref_name, 'SP/') || github.ref_name == 'master' )  &&
+    if: ${{ ( startsWith(github.ref_name, 'HF/')  || startsWith(github.ref_name, 'SP/') || github.ref_name == 'master' ) &&
```
Comment (kavitshah-gl) [inline .github/workflows/build.yml:102]: you can revert this change.

File: alfresco-googledrive-end-to-end-tests/pom.xml [hunk 3]
```diff
@@ -117,7 +117,7 @@
                                 </image>
                                 <image>
                                     <alias>activemq</alias>
-                                    <name>alfresco/alfresco-activemq:5.18.2-jre17-rockylinux8</name>
+                                    <name>alfresco/alfresco-activemq:5.17.4-jre17-rockylinux8</name>
```
Comment (kavitshah-gl) [inline alfresco-googledrive-end-to-end-tests/pom.xml:120]: Why are we downgrading the version?

**Review by kavitshah-gl — CHANGES_REQUESTED** [review]

**Review by kavitshah-gl — APPROVED** [review]
Approved

File: alfresco-googledrive-end-to-end-tests/pom.xml (hunk 3 above)
Comment (MohinishSah) [reply alfresco-googledrive-end-to-end-tests/pom.xml]: rebased the branch again and fixed

**Pull request merged**